python/ray/serve/tests/test_cli.py

```diff
@@ -713,5 +713,33 @@ def check_for_failed_deployment():
     wait_for_condition(check_for_failed_deployment, timeout=15)
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="File path incorrect on Windows.")
+def test_replica_placement_group_options(ray_start_stop):
+    """Test that placement group options can be set via config file."""
+
+    config_file_name = os.path.join(
+        os.path.dirname(__file__), "test_config_files", "replica_placement_groups.yaml"
+    )
+
+    subprocess.check_output(["serve", "deploy", config_file_name])
+
+    def check_application_status():
+        cli_output = subprocess.check_output(
+            ["serve", "status", "-a", "http://localhost:52365/"]
+        )
+        status = yaml.safe_load(cli_output)["applications"]
+        # TODO(zcin): fix error handling in the application state manager for
```

Comment:
@zcin here is the issue I discussed w/ you on slack. Plan to merge this as-is and you can take it as a follow-up.